### ISSUE TITLE
Add an output for the created pks_api_lb_security_group

### ISSUE THREAD

**Issue by dherbrich** [pivotal-cf/terraforming-aws#102]
HI Team.
in my scenario i want to use the PKS API over the internet. As the terraform creates an internet facing loadbalancer (with not option atm to create it as internal?) and the needed vm security group "pks_api_lb_security_group" everything is available for doing so. However i would like to automate my setup in pipelines and would like to use the terraform output to get the security group id to push it to bosh in a vm_extension.

In short: Please create an output for the resource "pks_api_lb_security_group" in the module and in the output of "terraforming-pks". https://github.com/pivotal-cf/terraforming-aws/blob/master/modules/pks/lb.tf#L2

**Comment by akirasoft** [comment]
I've bumped into this as well, took me longer than I'd like to admit to figure out why I couldn't access my pks api endpoint via the load balanced endpoint...

**Comment by adam-power** [comment]
I've also bumped into this, I created a pull request to resolve it: https://github.com/pivotal-cf/terraforming-aws/pull/134.

**Comment by nmahoney-pivotal** [comment]
Just merged #134
